pages/notebook/index.tsx

```diff
@@ -0,0 +1,68 @@
+/* 
+  The main purpos of this file is to redirect user to their default
+  organization notebook.
+*/
+import nookies from "nookies";
+import NextError from "next/error";
+import { AUTH_TOKEN } from "~/config/constants";
+import { generateApiUrl } from "~/config/api";
+import { fetchUserOrgs } from "~/config/fetch";
+import { captureEvent } from "~/config/utils/events";
+
+interface Props {
+  errorCode: number;
+}
+
+const NotebookPage = ({ errorCode }: Props) => {
+  if (errorCode) {
+    return <NextError statusCode={errorCode} />;
+  }
+
+  return null;
+};
+
+export async function getServerSideProps(ctx) {
+  const cookies = nookies.get(ctx);
+  const authToken = cookies[AUTH_TOKEN];
+
+  if (!authToken) {
```

Comment:
How/when do you think it will happen? I'm not sure I understand the scenario that might result in it failing.
Could there be a scenario where user is logged in but cookies are not set?

Reply:
I see. Do you think it is best then to make a request on the BE to some endpoint like the one that gets user info and then examine if it results in a 401?